### ISSUE TITLE
The file picker is not installed. Check our Getting Started documentation to install it.

### ISSUE THREAD

**Issue by mikevercoelen** [GetStream/stream-chat-react-native#2113]
I have installed all (optional) dependencies as listed in the Getting Started documentation. 

Getting the following error on `pickFile`

```
 The file picker is not installed. Check our Getting Started documentation to install it.
```

It's pretty unclear why this is happening, what do I do? This should be documented properly.

**Comment by mikevercoelen** [comment]
Ah great, I'll work with a patch for now then!

**Comment by vanGalilea** [comment]
Solved with in [v5.15.0](https://github.com/GetStream/stream-chat-react-native/releases/tag/v5.15.0)
